docs/essentials/exceptions/exceptions-best-practices.md

```diff
@@ -0,0 +1,319 @@
+# Best practices for exceptions
+
+A well-designed app handles exceptions and errors to prevent app crashes. This article describes best practices for handling and creating exceptions.
+
+## Handle exceptions
+
+The following list contains some general guidelines for handling exceptions in your app.
+
+- Use exception handling code (**try**/**catch** blocks) appropriately. You can also programmatically check for a condition that is likely to occur without using exception handling.
+
+  **Programmatic checks**. The following example uses an **if** statement to check whether a connection is closed. If it isn't, the example closes the connection instead of throwing an exception.
+
+  C#
+  ```
+  if (conn.State != ConnectionState.Closed)
+  {
+      conn.Close();
+  }
+  ```
+
+  C++
```

Comment:
I don't think we should show examples with languages that are not supported with .NET Core like C++ or VB (for now). We also should make sure that when we're porting the content, the samples we're showing use classes that are supported in .NET Core IMO.